gym/vector/vector_env.py

```diff
@@ -57,3 +57,65 @@ def __del__(self):
         if hasattr(self, 'closed'):
             if not self.closed:
                 self.close()
+
+
+class VectorEnvWrapper(VectorEnv):
+    r"""Wraps the vectorized environment to allow a modular transformation. 
+    
+    This class is the base class for all wrappers for vectorized environments. The subclass
+    could override some methods to change the behavior of the original vectorized environment
+    without touching the original code. 
+    
+    .. note::
+    
+        Don't forget to call ``super().__init__(env)`` if the subclass overrides :meth:`__init__`.
+    
+    """
+    def __init__(self, env):
+        assert isinstance(env, VectorEnv)
+        self.env = env
+        self.metadata = env.metadata
+        self.num_envs = env.num_envs
+        self.observation_space = env.observation_space
+        self.action_space = env.action_space
+        self.single_observation_space = env.single_observation_space
+        self.single_action_space = env.single_action_space
+        self.reward_range = env.reward_range
+
+    def __getattr__(self, name):
+        if name.startswith('_'):
+            raise AttributeError("attempted to get missing private attribute '{}'".format(name))
+        return getattr(self.env, name)
+
+    @property
+    def spec(self):
+        return self.env.spec
+
+    @classmethod
+    def class_name(cls):
+        return cls.__name__
```

Comment:
I know there is a similar method in the Wrapper class, but it seems very redundant (in both cases)